libevm/interop/invoke_test.go

```diff
@@ -115,6 +115,153 @@ func TestInvoke(t *testing.T) {
 	call(t, instance, "CloseDatabase", lib.DatabaseParams{DatabaseHandle: dbHandle})
 }
 
+func TestInvokeDelegatecall(t *testing.T) {
```

Comment:
We don't really need to go through the interop mechanism for this test. We could move it to the `lib` package to improve readability. Instead of `call(t, instance, "StateSetNonce", ...)` we would have `instance.StateSetNonce(...)`.